qualtran/bloqs/mod_arithmetic/mod_multiplication.py

```diff
@@ -656,7 +660,13 @@ def on_classical_vals(
         if self.uncompute:
             assert (
                 target is not None
-                and target == (x * y * pow(2, self.bitsize * (self.mod - 2), self.mod)) % self.mod
+                and target
+                == (
+                    x
+                    * y
+                    * pow(2, (self.bitsize + self._add_extra_qubit) * (self.mod - 2), self.mod)
+                )
+                % self.mod
```

Comment:
can you refactor this so it isn't formatted as badly. One possibility would be to refactor into an `if x: raise AssertionError`